### PR TITLE
feat: add milestones

### PR DESCRIPTION
**Pull request by LiteSun**
Fixes: #[Add issue number here]

Changes:
feat: add milestones in header and footer

Screenshots of the change:
image
image

### CODE REVIEW

**Comment by guoqqqi** [comment]
Can you add some before and after pictures of the changes as there is no preview link？

**Review by yzeng25 — DISMISSED** [review]

File: blog/src/theme/Footer/index.tsx [hunk 1]
```diff
@@ -92,6 +92,11 @@ const footer = {
           to: '/plugins',
           target: '_parent',
         },
+        {
+          label: 'Milestones',
```
Comment (guoqqqi) [inline blog/src/theme/Footer/index.tsx]:
```suggestion
          label: 'Roadmap',
```

File: config/navbar.js [hunk 2]
```diff
@@ -106,6 +106,11 @@ module.exports = [
         label: 'Events',
         target: '_parent',
       },
+      {
+        to: 'https://github.com/apache/apisix/milestones',
+        label: 'Milestones',
```
Comment (guoqqqi) [inline config/navbar.js]:
```suggestion
        label: 'Roadmap',
```

File: doc/src/theme/Footer/index.tsx [hunk 3]
```diff
@@ -92,6 +92,11 @@ const footer = {
           to: '/plugins',
           target: '_parent',
         },
+        {
+          label: 'Milestones',
```
Comment (guoqqqi) [inline doc/src/theme/Footer/index.tsx]:
```suggestion
          label: 'Roadmap',
```

File: website/src/theme/Footer/index.tsx [hunk 4]
```diff
@@ -89,6 +89,11 @@ const footer = {
           to: '/plugins',
           target: '_parent',
         },
+        {
+          label: 'Milestones',
```
Comment (guoqqqi) [inline website/src/theme/Footer/index.tsx]:
```suggestion
          label: 'Roadmap',
```

**Review by guoqqqi — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Pull request merged**